[ISSUE] - Crash when use ME Pattern Encoding Terminal in the server

When I was playing version 0.7.3, I set up a server to play, but when I started using the ME Pattern Encoding Terminal to make a Crafting Pattern, the server would most likely crash abruptly.

[21:55:07 INFO]: Unity[/127.0.0.1:58152] logged in with entity id 1794 at ([world]30.658734102019107, 80.0, 12.420468159110612)
[21:56:13 INFO]: Constructed recipe list for minecraft:stonecutting in 1.569 ms. 3354/3354 recipes are parallelized.
[21:56:17 ERROR]: Encountered an unexpected exception
java.lang.NoClassDefFoundError: net/minecraft/client/Minecraft
        at lu.kolja.expandedae.helper.misc.KeybindUtil.isKeyDown(KeybindUtil.java:21) ~[expandedae-1.2.0-cosmolite.6.jar%23851!/:1.2.0-cosmolite.6]
        at lu.kolja.expandedae.helper.misc.KeybindUtil.isShiftDown(KeybindUtil.java:9) ~[expandedae-1.2.0-cosmolite.6.jar%23851!/:1.2.0-cosmolite.6]
        at appeng.menu.me.items.PatternEncodingTermMenu.handler$cio000$encode(PatternEncodingTermMenu.java:1323) ~[appliedenergistics2-forge-15.4.7-cosmolite.8.jar%23860!/:?]
        at appeng.menu.me.items.PatternEncodingTermMenu.encode(PatternEncodingTermMenu.java:303) ~[appliedenergistics2-forge-15.4.7-cosmolite.8.jar%23860!/:?]
        at appeng.menu.AEBaseMenu$ClientAction.handle(AEBaseMenu.java:1000) ~[appliedenergistics2-forge-15.4.7-cosmolite.8.jar%23860!/:?]
        at appeng.menu.AEBaseMenu.receiveClientAction(AEBaseMenu.java:898) ~[appliedenergistics2-forge-15.4.7-cosmolite.8.jar%23860!/:?]
        at appeng.core.sync.packets.GuiDataSyncPacket.serverPacketData(GuiDataSyncPacket.java:74) ~[appliedenergistics2-forge-15.4.7-cosmolite.8.jar%23860!/:?]
        at appeng.core.sync.network.NetworkHandler.lambda$serverPacket$8(NetworkHandler.java:77) ~[appliedenergistics2-forge-15.4.7-cosmolite.8.jar%23860!/:?]
        at net.minecraftforge.network.NetworkEvent$Context.enqueueWork(NetworkEvent.java:210) ~[forge-1.20.1-47.4.6-universal.jar%23720!/:?]
        at appeng.core.sync.network.NetworkHandler.serverPacket(NetworkHandler.java:74) ~[appliedenergistics2-forge-15.4.7-cosmolite.8.jar%23860!/:?]
        at appeng.core.sync.network.__NetworkHandler_serverPacket_ClientCustomPayloadEvent.invoke(.dynamic) ~[appliedenergistics2-forge-15.4.7-cosmolite.8.jar%23860!/:?]
        at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.5.jar%23131!/:?]
        at net.minecraftforge.eventbus.EventBus.post(EventBus.java:319) ~[eventbus-6.0.5.jar%23131!/:?]
        at net.minecraftforge.eventbus.EventBus.post(EventBus.java:299) ~[eventbus-6.0.5.jar%23131!/:?]
        at net.minecraftforge.network.NetworkInstance.dispatch(NetworkInstance.java:67) ~[forge-1.20.1-47.4.6-universal.jar%23720!/:?]
        at net.minecraftforge.network.NetworkHooks.lambda$onCustomPayload$1(NetworkHooks.java:88) ~[forge-1.20.1-47.4.6-universal.jar%23720!/:?]
        at java.util.Optional.map(Optional.java:260) ~[?:?]
        at net.minecraftforge.network.NetworkHooks.onCustomPayload(NetworkHooks.java:88) ~[forge-1.20.1-47.4.6-universal.jar%23720!/:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.m_7423_(ServerGamePacketListenerImpl.java:2959) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.network.protocol.game.ServerboundCustomPayloadPacket.m_5797_(ServerboundCustomPayloadPacket.java:37) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.network.protocol.game.ServerboundCustomPayloadPacket.m_5797_(ServerboundCustomPayloadPacket.java:10) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.network.protocol.PacketUtils.m_263899_(PacketUtils.java:25) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:151) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:967) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:175) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:124) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.server.MinecraftServer.m_129961_(MinecraftServer.java:950) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.server.MinecraftServer.m_7245_(MinecraftServer.java:944) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.util.thread.BlockableEventLoop.m_18701_(BlockableEventLoop.java:134) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.server.MinecraftServer.mixinextras$bridge$m_18701_$269(MinecraftServer.java) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.server.MinecraftServer.wrapOperation$zdb000$managedBlock(MinecraftServer.java:2164) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.server.MinecraftServer.m_130012_(MinecraftServer.java:930) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:855) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:285) ~[server-1.20.1-20230612.114412-srg.jar%23715!/:?]
        at java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.Minecraft
        at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:157) ~[securejarhandler-mohist-2.1.11.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
        ... 37 more
[21:56:17 FATAL]: Preparing crash report with UUID 3d2b5515-73f3-4348-9806-8a160754cc77

--- a/src/main/java/lu/kolja/expandedae/mixin/terminal/MixinPatternEncodingTerminalMenu.java
+++ b/src/main/java/lu/kolja/expandedae/mixin/terminal/MixinPatternEncodingTerminalMenu.java
@@ -12,10 +12,8 @@ import appeng.menu.me.common.MEStorageMenu;
 import appeng.menu.me.items.PatternEncodingTermMenu;
 import appeng.menu.slot.RestrictedInputSlot;
 import de.mari_023.ae2wtlib.wut.WUTHandler;
-import java.util.Objects;
 import java.util.concurrent.atomic.AtomicReference;
 import lu.kolja.expandedae.definition.ExpItems;
-import lu.kolja.expandedae.helper.misc.KeybindUtil;
 import net.minecraft.world.entity.player.Inventory;
 import net.minecraft.world.entity.player.Player;
 import net.minecraft.world.inventory.MenuType;
@@ -51,14 +49,6 @@ public abstract class MixinPatternEncodingTerminalMenu extends MEStorageMenu imp
         AtomicReference<Player> player = new AtomicReference<>();
         this.getActionSource().player().ifPresent(player::set);
         if (encodedPatternSlot.getItem() != ItemStack.EMPTY) {
-            if (KeybindUtil.isShiftDown()) {
-                if (player.get().getInventory().getFreeSlot() > 0) {
-                    player.get().addItem(encodedPatternSlot.getItem());
-                    encodedPatternSlot.set(ItemStack.EMPTY);
-                    encodedPatternSlot.setChanged();
-                }
-            }
-
             var terminalItem = expandedae$getTerminalItem(player.get());
             if (terminalItem == null) return;
             if (terminalItem.getItem() instanceof IUpgradeableItem item) {
@@ -68,7 +58,7 @@ public abstract class MixinPatternEncodingTerminalMenu extends MEStorageMenu imp
 
             var blankPatternSlotCount = blankPatternSlot.getItem().getCount();
             if (node == null) return;
-            int changed = (int) Objects.requireNonNull(node).getGrid().getStorageService().getInventory().extract(
+            int changed = (int) node.getGrid().getStorageService().getInventory().extract(
                     AEItemKey.of(AEItems.BLANK_PATTERN),
                     64 - blankPatternSlotCount,
                     Actionable.MODULATE,

